woocommerce/woocommerce-gutenberg-products-block
# Filter by Rating: Add E2E tests

This PR adds E2E tests for the **Filter by Rating** filter block.

Closes #7185

#### Automated Tests

#### Testing
1. Run `npm run wp-env start`
2. Run: 
- `npm run test:e2e tests/e2e/specs/backend/rating-filter.test.js`
- `npm run test:e2e tests/e2e/specs/shopper/filter-products-by-rating.test.ts`
3. Make sure they pass without errors


### WooCommerce Visibility
